vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -21,111 +21,132 @@
 #include <host/functions.h>
 #include <host/pkg.h>
 #include <misc/cpp/imgui_stdlib.h>
+#include <rif2zrif.h>
 #include <util/log.h>
 
 #include <nfd.h>
 
 namespace gui {
 
-nfdchar_t *pkg_path = nullptr;
+static nfdchar_t *pkg_path, *work_path = nullptr;
+static std::string state, title, zRIF;
+static bool draw_file_dialog = true;
+static bool delete_pkg_file, delete_work_file = false;
```

Comment:
desgroup asked not to use static variables in the gui, that is why they are in state.h

Reply:
Set this inside state no have sense, is only for pkg dialog, and use static here is absoluly not problème